yarn.lock

```diff
@@ -7947,6 +8143,11 @@ typescript-type-utils@^0.1.0:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.3.tgz#3aea307c1746b8c384435d8ac36b8a2e580d85db"
   integrity sha512-CIfGzTelbKNEnLpLdGFgdyKhG23CKdKgQPOBc+OUNrkJ2vr+KSzsSV5kq5iWhEQbok+quxgGzrAtGWCyU7tHnA==
 
+typescript@~4.8.4:
```

Comment:
Let's avoid adding the old version back into repo

Reply:
unfortunately @microsoft/api-extractor uses this version at the moment